test/integration/offline.js

```diff
@@ -20,6 +20,10 @@ let docs = [
   {group: 'B', path: '/a/b/d', year: 2104}
 ]
 
+function sleep (ms) {
```

Comment:
Maybe we can use the `sleep ()` function in [utils.js](https://github.com/cozy/cozy-client-js/blob/master/src/utils.js).